lnwire/neighbor_ack.go

```diff
@@ -30,7 +31,9 @@ func (msg *NeighborAckMessage) Decode(r io.Reader, pver uint32) error {
 }
 
 func (msg *NeighborAckMessage) MaxPayloadLength(uint32) uint32 {
-	return 0
+	// Some random number. Transmission functions work bad if it is 0
```

Comment:
The number returned here should be exactly the same size as the message written above. So if you choose to switch to writing a single byte, then `1` should be returned, otherwise `18`.